static/js/all-alerts.js

```diff
@@ -23,9 +23,10 @@ let alertGridDiv = null;
 let alertRowData = [];
 
 let mapIndexToAlertState=new Map([
-    [0,"Normal"],
-    [1,"Pending"],
-    [2,"Firing"],
+    [0, "Inactive"],
+    [1,"Normal"],
+    [2,"Pending"],
+    [3,"Firing"],
```

Comment:
What does `Inactive` vs `Normal` mean?

Reply:
Inactive is the initial state when the alert was first created. It also implies, that the alert conditions evaluations has not been executed even once.

Reply:
Ok and we want to distinguish between them so the user can see which alerts haven't run vs which alerts have run and are normal?

I see that `evaluateMinionSearch()` sometimes uses a combination like this:
```
err := updateMinionSearchStateAndCreateAlertHistory(msToEvaluate, alertutils.Inactive, alertutils.AlertNormal)
```
Do we need to change that so it's just using one of the them? (If we do need to change that, it can be a separate PR  if you want)

Reply:
I have updated it in this PR, itself. Now Minion search also updates the state to `Normal` instead of `Inactive`.